go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/trustwallet/blockatlas
 go 1.12
 
 require (
+	github.com/davecgh/go-spew v1.1.1
```

Comment:
`go-spew` not used?

Reply:
It's used in the tests. I don't like `go.mod` / `go.sum` it pollutes the commits. Maybe we should remove it.
@kolya182 You can use `export GO111MODULE=off` to disable go.mod if you want